docs/forge.org

```diff
@@ -807,9 +807,8 @@ a post visits that post in a browser.
 
 Creating a new post and editing an existing post are similar to
 creating a new commit and editing the message of an existing commit.
-In both cases the message has to be written in a separate buffer.
-And then the process has to be finished or canceled using a separate
-command.
+In both cases the message has to be written in a separate buffer, then
+the process has to be finished or canceled using a separate command.
```

Comment:
I am aware that sentences should not begin with `and`, but have decided I don't care. Still, I can see why it would make to combine these two sentences into one. How about doing that, while keeping the `and`?